packages/inspection-capture-web/src/PhotoCapture/PhotoCaptureHUD/PhotoCaptureHUDTutorial/PhotoCaptureHUDTutorial.tsx

```diff
@@ -0,0 +1,129 @@
+import { CSSProperties } from 'react';
+import { useTranslation } from 'react-i18next';
+import { Button, DynamicSVG } from '@monkvision/common-ui-web';
+import { CaptureAppConfig } from '@monkvision/types';
+import { styles } from './PhotoCaptureHUDTutorial.styles';
+import { TutorialViews } from '../../hooks';
+import { usePhotoCaptureHUDButtonBackground } from '../hooks';
+import { SightGuideline } from '../PhotoCaptureHUDElementsSight';
+import { arrowGuidelineSVG, arrowSightTutorialSVG } from '../../../assets';
+
+const TRANSLATION_PREFIX = 'photo.hud.tutorial';
+
+/**
+ * Props of the PhotoCaptureHUDTutorial component.
+ */
+export interface PhotoCaptureHUDTutorialProps
+  extends Pick<
+    CaptureAppConfig,
+    'enableSightGuidelines' | 'allowSkipTutorial' | 'sightGuidelines' | 'enableSightTutorial'
+  > {
+  /**
+   * The id of the sight.
+   */
+  sightId: string;
+  /**
+   * The current tutorial view in PhotoCapture component.
+   */
+  tutorialView: TutorialViews | null;
+  /**
+   * Callback called when the user clicks on "Next" button.
+   */
+  onTutorialNext: (view: TutorialViews | null) => void;
+}
+
+function getButtonStyle(enableAddDamage?: boolean): CSSProperties {
+  return { visibility: enableAddDamage ? 'visible' : 'hidden' };
+}
+
+function ArrowIcon({ tutorialView }: Pick<PhotoCaptureHUDTutorialProps, 'tutorialView'>) {
+  if (tutorialView === TutorialViews.GUIDELINE) {
+    return <DynamicSVG style={styles['arrowGuideline']} svg={arrowGuidelineSVG} />;
+  }
+  if (tutorialView === TutorialViews.SIGHT_TUTORIAL) {
+    return <DynamicSVG style={styles['arrowSightTutorial']} svg={arrowSightTutorialSVG} />;
+  }
+  return null;
+}
+
+function DisplayText({ tutorialView }: Pick<PhotoCaptureHUDTutorialProps, 'tutorialView'>) {
+  const { t } = useTranslation();
+
+  const textArray = t(`${TRANSLATION_PREFIX}.${tutorialView}`).split(/<br\s*\/?>/);
+
+  return (
+    <div style={styles['text']}>
+      <div style={styles['title']} data-testid='title'>
+        {t(`${TRANSLATION_PREFIX}.title`)}
+      </div>
+      {textArray.map((value: string, index: number) => (
+        <div key={index}>
+          {value}
+          {index < textArray.length - 1 && <br />}
+        </div>
+      ))}
+    </div>
+  );
+}
```

Comment:
Create a directory + index.ts + TS file for each of these components. Only one component per file!